src/components/organisms/EntityDetails/EntityDetailsContent/ExecutionsTable/EmptyExecutionsListContent/EmptyExecutionsListContent.tsx

```diff
@@ -15,11 +17,28 @@ const EmptyExecutionsListContent: React.FC<EmptyExecutionsListContentProps> = pr
 
   const {entity, entityDetails} = useContext(EntityDetailsContext);
   const {dispatch} = useContext(MainContext);
+  const mayRun = usePermission(Permissions.runEntity);
 
   if (!entityDetails) {
     return null;
   }
 
+  if (!mayRun) {
+    return (
+      <EmptyListContent
+        title="No executions found"
+        description={
+          <>
+            Your {entity === 'tests' ? 'test' : 'test suite'} has no past executions. We will update this list as soon
+            as the first execution was started. <br />
+            Since you do only have read permissions on this environments you can not trigger a test/test suite run
```

Comment:
Here we might also want to change wording

Reply:
It should not happen on OSS, so it's mostly to align the code and ensure that it will not fail.

I believe that we could think of changing the texts when there will be Cloud and OSS merged, will take a note on that 👍